README.md

```
@@ -1,38 +1,174 @@
+
+GitHub Actions를 사용한 자동 배포 파이프라인
 
-- Spring Boot 3.x
-- Java 21
-- Spring Data JPA
-- PostgreSQL
-- Swagger
-- AWS EC2 / S3
+| 브랜치 | 환경       | 설명               |
+| ------ | ---------- | ------------------ |
+| `dev`  | Development | 개발 서버          |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe does not align with header for style "aligned"

(MD060, table-column-style)